### DISCUSSION TITLE
AngularJS Documentation gone?

### DISCUSSION THREAD

**Discussion by jaderanderson** [l-lin/angular-datatables#1539] [Q&A] [answered]
Hello all, 

I know this repo migrated to Angular and now the AngularJS part is gone , but why purge the AngularJS documentation?

Its possible to generate at least a dump .md file with all the examples?

We still use the original angularJS component while the new tool is being constructed on modern tech, but deleting the old docs leave us blind to troubleshoot or implement modifications on behaviors already included.

**Comment by shanmukhateja** [comment, marked as answer]
Hi, you can find the older documentation in the `angular1` branch [here](https://github.com/l-lin/angular-datatables/tree/angular1)

**Reply by jaderanderson** [reply]
Thank you so much!
